proposals/module-types/Explainer.md

```diff
@@ -244,7 +244,15 @@ Just as with module definitions, the above module type is actually an
   (export "d" (func (result f32)))
 )
 ```
-which is the form we'll mostly use in this document.
+which is primarily used in this proposal.
+
+Although the text and binary format list imports and exports in a particular
+order, module types describe their imports and exports via *unordered maps*
+and thus two encoded module types that differ only by order are equivalent.
```

Comment:
`instantiate` only uses the order of the *local* module-type declaration.  You can think of it as a binary encoding scheme that allows `instantiate` to avoid duplicating the sequence of names in-inline by pointing to a module typedef.

Reply:
I understand, but I'd argue that it's still somewhat misleading. The order matters in some contexts but not in others. Plus, it is interconvertible. But it's more of a nit.

Reply:
Ok, so would you say instead that module types do imply order and there's just subtyping between all the different orderings?

Reply:
Yeah, that would be a more accurate description.

Reply:
Great, done.